packages/grafana-ui/src/utils/table.ts

```diff
@@ -34,5 +34,5 @@ export const getCellLinks = (field: Field, row: Row) => {
     }
   }
 
-  return links;
+  return links.filter((link) => link.href || link.onClick != null);
```

Comment:
bit of a smell to me that we do it two places, maybe a candidate for some cleanup at some point.